sigma/pipelines/panther/panther_sdyaml_pipeline.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from typing import Optional, List
+from sigma.processing.conditions import FieldNameProcessingCondition
+from sigma.processing.pipeline import ProcessingItem, ProcessingPipeline
+from sigma.processing.transformations import (
+    ReplaceStringTransformation,
+    FieldMappingTransformationBase,
+)
+from sigma.rule import SigmaDetectionItem
+
+
+@dataclass
+class AddFieldnamePrefixAndSuffixTransformation(FieldMappingTransformationBase):
+    """
+    Add field name prefix.
+    """
+
+    prefix: str

```

Comment:
Would it make sense to set defaults for the `prefix` and `suffix` attributes?

E.g.,
```py
prefix: str = ""
suffix: str = ""
```